frontend/assets/css/colors.scss

```
@@ -124,6 +127,19 @@
 
   --toggle-button-slider-color: var(--light-grey);
 
+  --toast-message-success-color: var(--green);
```

nitpick: [bem](https://getbem.com/)-like syntax would be easier to read

- stick to browser names for things, as far as possible
- double dashes for modifiers (in BEM words)

```suggestion
  --toast-message-background-color--success: var(--green);
```